src/core/storage/disk.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import type {ByteString, WritableDataStorage} from "./index";
+import {join as pathJoin, normalize} from "path";
+import fs from "fs-extra";
+
+/**
+ * Disk Storage abstracts away low-level file I/O operations, implementing the
+ * Writable Data Storage interface. In this implementation, `path` aliases to
+ * a key, and the `contents` parameter maps to the value.
```

Comment:
> In this implementation, `path` aliases to a key, and the `contents`
> parameter maps to the value.

What does this mean? What is aliasing, what does it mean to “alias to a
key”, and what does it mean to “map to the value”? Are these doc
comments better suited for specific methods? (The `DiskStorage` type
itself does not have `path` or `contents` parameters.) I’m not sure what
you’re trying to convey.